stablehlo/reference/Tensor.cpp

```diff
@@ -370,13 +373,53 @@ IndexSpaceIterator Tensor::index_begin() const {
 
 IndexSpaceIterator Tensor::index_end() const { return getShape().index_end(); }
 
-void Tensor::print(raw_ostream &os) const {
-  getType().print(os);
-  os << " {\n";
+namespace {
+
+void printNewline(llvm::raw_ostream &os, int64_t n) {
```

Comment:
`printIndentation` may be ... as this is doing more than printing newlines.

Reply:
went with `printNewlineIndent`